### ISSUE TITLE
100% crash on Kinemation Getting Started Sample (XBot from Mixamo)

### ISSUE THREAD

**Issue by gabrieldechichi** [Dreaming381/Latios-Framework-Documentation#3]
Hey - sorry to come back already with a couple of issues, but I'm getting 100% crash when trying to implement Kinemation's Getting Started examples.

```
Unity version: 2022.3.1f1
Latios version: tested both 0.7.5 and 0.7.3
Model: XBot from Mixamo
```

Steps:
- Import XBot and setup according to Kinemation's documentation (using the `LatiosDeform` node)
- Import a Walking clip from Mixamo
- Implement any of the examples (non-optimized, non-optimized per bone, or optimized skeleton)
- Hit play and wait a couple of seconds

It also happens just by hitting play and trying to move the model (no animation). Happens with subscene closed and open. When the subscene is open, LatiosTransform gives warnings about missing childs.

[crash-subscene-closed.log](https://github.com/Dreaming381/Latios-Framework-Documentation/files/11944385/crash-subscene-closed.log)

[crash-subscene-open.log](https://github.com/Dreaming381/Latios-Framework-Documentation/files/11944387/crash-subscene-open.log)

![image](https://github.com/Dreaming381/Latios-Framework-Documentation/assets/15620434/5278ca1e-fb33-416f-b46e-5324e2356a21)

**Comment by gabrieldechichi** [comment]
Uploading the project assets for the crash as well.

[Crashdemo.zip](https://github.com/Dreaming381/Latios-Framework-Documentation/files/11944410/Crashdemo.zip)

**Comment by Dreaming381** [comment]
This is the jobs debugger crash regression that was introduced in 2022.2.20. At least one of the causes was fixed in 2022.3.3 so please switch to that version or newer. The proposed workaround prior to that was to disable the jobs debugger.

I will try to reproduce the missing children warning later this week.

**Comment by gabrieldechichi** [comment]
Thanks! That fixed it, warning for Latios does still appear but seems to be benign for now. Thanks!
